ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/port_channel_interfaces.py

```diff
@@ -63,7 +63,8 @@ def port_channel_interfaces(self) -> list | None:
             elif (vlans := link.get("vlans")) is not None:
                 port_channel_interface["vlans"] = vlans
 
-            if self.shared_utils.mlag is True:
+            # Configure MLAG on MLAG switches unless "mlag_on_orphan_port_channel_downlink" is False AND link.mlag is False
+            if self.shared_utils.mlag is True and (get(self._hostvars, "mlag_on_orphan_port_channel_downlink", default=True) or link.get("mlag", True)):
```

Comment:
This is tricky to link the comment and the boolean expressions underneath as all of them are inverted and the comment is incomple (it also requires `self.shared_utils.mlag` to be True to configure MLAG

Maybe using `any` would make it clearer (disclaimer this was handtyped in Github, python code may need adjusting)

```suggestion
        # Configure MLAG on MLAG switches if either 'mlag_on_orphan_port_channel_downlink' or 'link.mlag' is True
        if self.shared_utils.mlag is True and any([get(self._hostvars, "mlag_on_orphan_port_channel_downlink", default=True), link.get("mlag", True)]):
```